contracts/RiskFund/RiskFund.sol

```diff
@@ -233,7 +233,7 @@ contract RiskFund is AccessControlledV8, ExponentialNoError, ReserveHelpers, Max
      */
     function getPoolsBaseAssetReserves(address comptroller) external view returns (uint256) {
         require(ComptrollerInterface(comptroller).isComptroller(), "Risk Fund: Comptroller address invalid");
-        return poolsAssetsReserves[comptroller][convertibleBaseAsset];
+        return _poolsAssetsReserves[comptroller][convertibleBaseAsset];
     }
```

Comment:
Would it make sense to have something similar for the ProtocolShareReserves contract? In that, could we move this  function to the ReserveHelpers file?

Reply:
There's no notion of "base asset" in reserve helpers, so I think we should be fine with `getPoolAssetReserve`. `getPoolsBaseAssetReserves(comptroller)` is basically `getPoolsBaseAssetReserves(comptroller, convertibleBaseAsset)`

Reply:
But we may need sth like this in the _interface_ though

Reply:
My fault, I didn't take into account `ReserveHelpers.getPoolAssetReserve`. We can keep it as it is